File: blog-post-2/c/Factorial.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int number,factorial = 1,i;
    printf("Enter number: ");
    scanf("%d",&number);
    for (int i = 1; i <= number; i++)
    {
        factorial *= i;
    }
    printf("%d ! = %d",number,factorial);
    return 0;
}

File: blog-post-2/kotlin/Factorial.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var scanner = Scanner(System.`in`);
    print("Enter number: ");
    var number = scanner.nextInt();
    scanner.close();
    var factorial = 1;
    for(x in 1..number){
        factorial *= x;
    }
    println(number.toString() + "! = " + factorial);
}